.circleci/config.yml

```diff
@@ -1,25 +1,21 @@
 version: 2.1
 
-executors:
-  go:
-    docker:
-      - image: circleci/golang:1.14
-    environment:
-      GOMAXPROCS: 4
-      GO111MODULE: "on"
-      GOPROXY: https://proxy.golang.org/
-      TEST_RESULTS_DIR: &TEST_RESULTS_DIR /tmp/test-results
+orbs:
+  win: circleci/windows@2.2.0
 
-jobs:
-  go-checks:
-    executor:
-      name: go
+references:
+  environment: &ENVIRONMENT
+    GOMAXPROCS: 4
+    GO111MODULE: "on"
+    GOPROXY: https://proxy.golang.org/
+    TEST_RESULTS_DIR: &TEST_RESULTS_DIR /tmp/test-results
+    WIN_TEST_RESULTS: &WIN_TEST_RESULTS c:\Users\circleci\AppData\Local\Temp\test-results
+
+commands:
+  git-verify:
     steps:
-      - checkout
-      - run: go mod verify
-      - run: make fmt
-      - run:
-          name: verify no code was generated
```

Comment:
This is where I wrote a comment and GitHub seems to have deleted it for some reason, which is pretty rude. An attempt to reconstruct:

Looking at the test output, we're not running this command in the Linux tests. And in the Windows tests, we don't run `go mod verify` or `make fmt` before it, so it's always going to pass. It'd be great to have this check present and working on at least one of the platforms.

Reply:
Ahh good catch! I forgot to add that job to the workflows. I'll have the check job run before the tests on windows & linux.

Reply:
Fixed & squashed my commits- let me know how it looks! Can cherrypick once this is merged or open another PR (unless that's overkill). 

<img width="435" alt="Screen Shot 2020-04-21 at 6 13 19 PM" src="https://user-images.githubusercontent.com/12939567/79929832-cc8c5400-83fb-11ea-92a9-52193c2c4705.png">